ffi/dotnet/Devolutions.IronRdp.AvaloniaExample/App.axaml.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using Avalonia;
+using Avalonia.Controls.ApplicationLifetimes;
+using Avalonia.Markup.Xaml;
+
+namespace Devolutions.IronRdp.AvaloniaExample;
+
+public partial class App : Application
+{
+    public override void Initialize()
+    {
+        Console.WriteLine("App Initialize");
```

Comment:
question: We probably don’t really need such outputs in the terminal

Reply:
updated